### ISSUE TITLE
aws_cloudwatch_event_rule is_enabled flag is not working

### ISSUE THREAD

**Issue by stevesuh** [terraform-providers/terraform-provider-aws#10346]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.12.9
+ provider.archive v1.2.2
+ provider.aws v2.30.0

### Affected Resource(s)

* aws_cloudwatch_event_rule

### Terraform Configuration Files

```hcl
resource "aws_cloudwatch_event_rule" "retract_alarms" {
  name                = "${local.namespace}-schedule-every-8-hours"
  description         = "Triggers function once every 8 hours"
  schedule_expression = "cron(0 */8 * * ? *)"
  tags = local.tags
  is_enabled = false
}

```

### Expected Behavior

cloudwatch event rule should be disabled

### Actual Behavior

cloudwatch event rule is enabled

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

This was working in TF 0.12.6

### References

* #0000

**Comment by stevesuh** [comment]
I tested this again with aws provider v2.33.0.  Still not working.

**Comment by bflad** [comment]
Improved handling for the `is_enabled` argument has been merged and will release with version 2.44.0 of the Terraform AWS Provider, Thursday next week. Thanks to @DrFaust92 for the implementation.
